src/jit/lsraxarch.cpp

```diff
@@ -2151,25 +2137,17 @@ void LinearScan::TreeNodeInfoInitSIMD(GenTreeSIMD* simdTree)
                 // Need one SIMD register as scratch.
                 // See genSIMDIntrinsicRelOp() for details on code sequence generated and
                 // the need for one scratch register.
-                //
-                // Note these intrinsics produce a BOOL result, hence internal float
-                // registers reserved are guaranteed to be different from target
-                // integer register without explicitly specifying.
                 info->internalFloatCount = 1;
                 info->setInternalCandidates(this, allSIMDRegs());
             }
-            if (info->isNoRegCompare)
+            info->dstCount = 0;
+            // Codegen of SIMD (in)Equality uses target integer reg only for setting flags.
```

Comment:
This comment needs updating, it still mentions the target register even though it is never used.